tests/functional/test_request_pages.py

```diff
@@ -133,12 +133,18 @@ def test_request_return(live_server, context, page, bll):
         "file2.txt",
         "file 2 content",
     )
-    factories.bll.set_status(release_request, RequestStatus.SUBMITTED, author)
+    bll.set_status(release_request, RequestStatus.SUBMITTED, author)
 
     return_request_button = page.locator("#return-request-button")
+    page.goto(live_server.url + release_request.get_url())
+
+    def _logout():
```

Comment:
could we just clear the browser cookies here? Same effect, but avoids a couple of navigations and page loads?

Reply:
Maybe, I'll give it a try